node-manager/src/logging.rs

```diff
@@ -0,0 +1,32 @@
+use chrono::Utc;
+use lightning::util::logger::{Logger, Record, Level};
+use log::*;
+
+#[derive(Default, Debug, Eq, PartialEq, Copy, Clone)]
+pub struct MutinyLogger {}
```

Comment:
was going to point out `pub` here but you got to it before me. Tested it locally, works well in the browser.